firefox-ios/Client/metrics.yaml

```diff
@@ -2715,25 +2715,6 @@ firefox_home_page:
       - fx-ios-data-stewards@mozilla.com
     expires: "2025-07-01"
 
-# HomePage - General
-homepage:
-  private_mode_toggle:
```

Comment:
@ih-codes we are removing the private mask toggle so I removed the telemetry

cc: @afurlan-firefox